src/models/storage.ts

```diff
@@ -39,12 +42,42 @@ class EntryStorage {
     return newData;
   }
 
+  private static isOTPStorage(entry: object) {
+    const properties = [
+      'account', 'hash', 'index', 'issuer', 'type', 'counter', 'secret',
```

Comment:
I feel like I shouldn't have to manually type this out, but apparently when ts is compiled it loses all type definitions

Reply:
It's okay. TypeScript is syntax sugar for JavaScript, it cannot handle everything. Hardcode here is fine.